dct_backend/dct.py

```diff
@@ -30,3 +34,54 @@ def save_food_item(self, name: str, station: str, meal_time: str,
         item = FoodItem(name, station, meal_time, calories, categories)
         self.menu.append(item)
         return item
+
+    """
+        incoming_dct_traffic adds people who are scanning into the dct
+
+        takeout: int
+            The current number of takeout scanning in
+
+        dinein: int
+            The current number of dinein scanning in
+
+        returns: Capacity
+            The total number of takeout, dineine, and total capacity
+            of takeout + dinein
+    """
+    def incoming_dct_traffic(self, takeout: int, dinein: int) -> Capacity:
+        self.takeout_capacity += takeout
+        self.dinein_capacity += dinein
+        incomingtraffic = takeout + dinein
+        self.total_capacity += incomingtraffic
+        capacity = Capacity(takeout, dinein, incomingtraffic)
+        return capacity
+
+    """
+        outgoing_dct_traffic subtracts people who are leaving the dct
+
+        takeout: int
+            The current number of takeout leaving
+
+        dinein: int
+            The current number of dinein leaving
+
+        returns: Capacity
+            The total number of takeout, dineine, and total capacity
+            of takeout + dinein
+    """
+    def outgoing_dct_traffic(self, takeout: int, dinein: int) -> Capacity:
```

Comment:
I mentioned this on one of your other PRs (but ignore this if you don't think it is necessary) maybe return true or false for success and failure

Reply:
Given that whatever is tracking this (maybe a camera) won't be the most accurate, I don't think it would be helpful to return a status in this case.

One thing you can do however to make this a little more readable is:
`self.<capacity> = max(self.<capacity> - <capacity>, 0)`

to skip the if, else statement chaining. Though, that chaining does allow for more custom error handling, I think we'll be mocking most of this for now so its probably better to stick with the readable format.

Reply:
I was assuming it would ideally be tracked from the data retrieved through the computers when students swipe their cards.